common/src/main/java/edu/snu/nemo/common/ir/vertex/LoopVertex.java

```diff
@@ -184,6 +188,16 @@ public void addDagOutgoingEdge(final IREdge edge) {
     return this.dagOutgoingEdges;
   }
 
+  /**
+   * Marks invariant edges with DuplicateDataProperty.
+   */
+  public void markInvariantEdge() {
+    nonIterativeIncomingEdges.forEach(((irVertex, irEdges) -> irEdges.forEach(irEdge -> {
+      irEdge.setProperty(DuplicateDataProperty.of(Pair.of(String.valueOf(loopDuplicateDataId), undecided)));
```

Comment:
Using `String.valueOf(loopDuplicateDataId)` violates the specification 'Left part of the pair indicates the edge id' in the comment of `DuplicateDataProperty.java`. How about using an additional execution property?

Property A = (loopDuplicateDataId, numCount)
Property B = (physicalEdgeId, numCount)